### DISCUSSION TITLE
Get Rid of Hunger/Thirst

### DISCUSSION THREAD

**Discussion by tenaz123** [DarkCastleMUD/DarkCastle#240] [Ideas]
Hunger/thirst are an annoyance that served a purpose when lots of people played, but everyone gets freedom at level 60, so it only serves to annoy lower level players.

**Comment by TehDreadPirateRoberts** [comment]
I personally still like having to eat and drink while levelling up a char. Yes it is an annoyance sort of... but it brings back nostalgia every time I level a char and need to eat salty fish.

**Reply by tenaz123** [reply]
Maybe there should be a nostalgia toggle then to turn on/off annoying things?  Eeating/drinking and other annoying things I'm probably forgetting.

**Reply by tenaz123** [reply]
Like item damage - annoyance with no benefit.  Looting coins - they should go straight to gold balance - everyone automates it anyway.
